README.md

```diff
@@ -16,6 +16,99 @@ Run the following commands to perform a release:
     $ mvn release:prepare release:perform
     $ git push --all && git push --tags
 
+## Parameter Store
+
+Parameter naming
+
+The parameter names are made from the following:
+* Project Name
+* Environment (dev/prod/sandbox/etc)
+* Date that the value was added in YYYYMMDD format
+* Parameter key
+Example: /weapon-x/dev/20190130/subject

```

Comment:
I'd remove the date aspect. That would allow the params to be discover-able by the system. For example on system spin up it could identify it's "folder" `weapon-x/dev` and load all variables under that as config or env variables, allowing an admin to simply update, remove, or add configuration without a redeploy.

Reply:
There was a reason that the date was being included, I think for versioning, but it's been awhile since I wrote this. I know there was discussion around it and what format that we should be using for the date.